commercetools-models/src/test/java/io/sphere/sdk/carts/CartLikeTest.java

```diff
@@ -0,0 +1,51 @@
+package io.sphere.sdk.carts;
+
+import io.sphere.sdk.json.SphereJsonUtils;
+import io.sphere.sdk.utils.MoneyImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import static io.sphere.sdk.models.DefaultCurrencyUnits.EUR;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CartLikeTest {
+
+    private Cart cartWithTaxes;
+    private Cart cartWithoutTaxes;
+
+    @Before
+    public void setUp() throws Exception {
+        cartWithTaxes = SphereJsonUtils.readObjectFromResource("cart-with-taxes.json", Cart.class);
+        cartWithoutTaxes = SphereJsonUtils.readObjectFromResource("cart-without-taxes.json", Cart.class);
+    }
+
```

Comment:
Regarding the mocking framework, it's just a decision that was took long time ago that in general you should be able to test the JVM SDK without mocking (which was not possible with the previous Play SDK). Also in many projects we found tests that ended up not testing anything because of a misuse of the mocking capability. All together we decided to eat our own dog food and not use mocking frameworks. Which in the SDK is not a big deal because mostly everything are IT. But in Sunrise and other places, it can become really annoying to test and I consider the mocking framework solution.

I "can" use Impl but it's really not an option because it's just a huge constructor with all possible values coming from the API, not useful at all.

The only solution, besides the mocking framework, is to auto-generate test doubles, but that's actually just an equivalent of mocking.